pkg/controller/install/add_openshift.go

```diff
@@ -5,7 +5,7 @@ import (
 )
 
 func init() {
-	platformPreInstallFuncs = append(platformPreInstallFuncs, openshift.EnsureMaistra)
-	platformPostInstallFuncs = append(platformPostInstallFuncs, openshift.EnsureOpenshiftIngress)
+	platformPreInstallFuncs = append(platformPreInstallFuncs, openshift.EnsureMaistra, openshift.EnsureConfigMapForCrt)
+	platformPostInstallFuncs = append(platformPostInstallFuncs, openshift.EnsureOpenshiftIngress, openshift.EnsureOpenshiftRegistry)
```

Comment:
I hate to do this to you, but I've completely changed this in master. Hopefully, it's simpler, but you'll need to rebase this PR.

Reply:
Okey i will rebase and change accordingly...